### DISCUSSION TITLE
Does prisma support postgres CASE conditional expression?

### DISCUSSION THREAD

**Discussion by motoya-k** [prisma/prisma#12748] [Q&A]
Hi,
I would like to run the following query with prisma.

```sql
SELECT
    CASE WHEN name is null THEN 'unkonwn'
        ELSE name
        END as name 
FROM users;
```

Please let me know if this query is supported by the Prisma method or if I have to use the $queryRaw method.

**Comment by austincrim** [comment]
Hey @motoyannn 👋 ,

You will need to use `$queryRaw` in this instance. Please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you would like to see us add this functionality to the Prisma Client.

**Comment by bsplosion** [comment]
See https://github.com/prisma/prisma/issues/13529 for the feature request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
